addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -61,175 +69,461 @@
  *
  * @author Kai Kreuzer
  * @author Volker Daube
+ * @author Helmut Lehmeyer
+ *         generic DPT Mapper
  * @since 0.3.0
- *
  */
 public class KNXCoreTypeMapper implements KNXTypeMapper {
 
-    static private final Logger logger = LoggerFactory.getLogger(KNXCoreTypeMapper.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(KNXCoreTypeMapper.class);
```

Comment:
Thought about, but then some static function can not log, [like this.](https://github.com/kgoderis/openhab2/pull/9/files/e96c5042fbc8ccea702ac758a7b5815c2f3ed5b1#diff-0a73a0644601e4bd4e031b0907472063R936)